.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 .target/*
```

```suggestion
target/*
```
Ось тут в тебе помилка)